manifests/package.pp

```diff
@@ -9,15 +15,4 @@
   if $::operatingsystem == 'gentoo' {
     include puppet::package::gentoo
```

Comment:
The Gentoo class will need some changes as well, but I'll handle it in a follow up commit